### PR TITLE
Add Iraq

### PR DESCRIPTION
**Pull request by camilavcoutinho**
Add Iraq configurations.

### CODE REVIEW

**Comment by camilavcoutinho** [comment]
@wdsrocha It is still missing the mobile rule, could you help me with that?

File: src/script/countries/IRQ.coffee [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+Phone = require('../Phone')
+PhoneNumber = require('../PhoneNumber')
+
+# For more info check:
+# https://www.numberingplans.com/?page=dialling&sub=areacodes
+# http://en.wikipedia.org/wiki/Local_conventions_for_writing_telephone_numbers#Brazil
```
Comment (lucasecdb) [inline src/script/countries/IRQ.coffee:7]:
```suggestion
```

**Review by lucasecdb — APPROVED** [review]

**Comment by jeffersontuc** [comment]
Added some test cases.

**Review by lucasecdb — APPROVED** [review]
nice! good job @jeffersontuc and @camilavcoutinho ✨

**Review by lbebber — APPROVED** [review]

**Review by ArthurSampaio — APPROVED** [review]

**Pull request merged**